content/news/2023.pml

```diff
@@ -54,6 +54,27 @@ Older News: <%=reflist%>
 </div>
 -->
 
+<!----------------------------------------------------------->
+<div class="news-item" id="member-autoware">
+  20 March 2023: The Autoware Foundation joins the seL4 Foundation
+</div>
+<div>
+  <a href = "https://www.autoware.org/">
+    <img src="/Foundation/Membership/LOGOS/autoware.svg"
+    style="width: 15%; float:right"
+    alt="Autoware Foundation" />
+  </a>
+  <p>
+  The seL4 Foundation welcomes <a href = "https://www.autoware.org/">the Autoware Foundation</a> as a member.
```

Comment:
I suggest having a link on "member" to the membership page, where we see all the logos and other members